float64 up_p_r
float64 up_p_l
float64 mid_p_r
float64 mid_p_l
float64 feet_p_r
float64 feet_p_l

#float64 up_v_r
#float64 up_v_l
#float64 mid_v_r
#float64 mid_v_l
#float64 feet_v_r
float64 feet_v_l

float64 neck_p
#float64 neck_v
float64 head_p
#float64 head_v

float64 pose_x
float64 pose_y
float64 pose_z
float64 rot_r
float64 rot_p
float64 rot_y

